Ports/opusfile/patches/0001-Recognize-SerenityOS-as-a-host-platform.patch

```diff
@@ -0,0 +1,22 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Kenneth Myhra <kennethmyhra@gmail.com>
+Date: Thu, 22 Dec 2022 19:01:48 +0100
+Subject: [PATCH] Recognize SerenityOS as a host platform
+
+---
+ config.sub | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/config.sub b/config.sub
+index 1d8e98bcee23a0421e4fafe9a6c9ac75180cff25..e1d397a3c3126a96936406663df0a363846a415e 100755
+--- a/config.sub
++++ b/config.sub
+@@ -1390,7 +1390,7 @@ case $os in
+ 	      | -powermax* | -dnix* | -nx6 | -nx7 | -sei* | -dragonfly* \
+ 	      | -skyos* | -haiku* | -rdos* | -toppers* | -drops* | -es* \
+ 	      | -onefs* | -tirtos* | -phoenix* | -fuchsia* | -redox* | -bme* \
+-	      | -midnightbsd*)
++	      | -midnightbsd* | -serenity*)
```

Comment:
`use_fresh_config_sub`?

Reply:
Absolutely. I need to be more aware of this option. Thanks!